lib/std/Build.zig

```diff
@@ -59,6 +59,7 @@ verbose_llvm_bc: ?[]const u8,
 verbose_cimport: bool,
 verbose_llvm_cpu_features: bool,
 reference_trace: ?u32 = null,
+ttyconf: debug.TTY.Config,
```

Comment:
This is a public API so let's make the name match the type:

```suggestion
tty_config: debug.TTY.Config,
```